components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1548,6 +1556,8 @@
 <nu hgrid="ne11np4" >  2.0e16 </nu>
 <nu hgrid="ne16np4" >  7.0e15 </nu>
 <nu hgrid="ne30np4" >  1.0e15 </nu>
+<nu dyn_target="preqx"   hgrid="ne45np4" > 3.0E+14 </nu>
+<nu dyn_target="theta-l" hgrid="ne45np4" > 3.5e-8 </nu>
```

Comment:
This is a little confusing, just want to make sure it is intentional:  this will be the first cubed-sphere configuration that is using the tensor HV?   

With tensor HV, the coefficient is grid independent, so we should need to set it for each resolution.  That is, the value 3.5e-8 should work with all grids.

Reply:
Good point. 
So Maybe I should just add a line like this?
  <nu dyn_target="theta-l"> 3.4e-8</nu>
That matches the value used for RRM, so it wouldn't affect that configuration.

Reply:
After some offline discussion I decided to leave it in for now, and maybe later fix up this section to have a standard scaling anytime the tensor hyperviscosity is used.